torch/_torch_docs.py

```diff
@@ -7810,12 +7810,18 @@ def merge_dicts(*dicts):
            r"""
 sgn(input, *, out=None) -> Tensor
 
-For complex tensors, this function returns a new tensor whose elemants have the same angle as that of the
-elements of :attr:`input` and absolute value 1. For a non-complex tensor, this function
-returns the signs of the elements of :attr:`input` (see :func:`torch.sign`).
+Returns a new tensor with the signs (definition below) of the elements of :attr:`input`.
 
-:math:`\text{out}_{i} = 0`, if :math:`|{\text{{input}}_i}| == 0`
-:math:`\text{out}_{i} = \frac{{\text{{input}}_i}}{|{\text{{input}}_i}|}`, otherwise
+.. math::
+    \text{out}_{i} = \begin{cases}
+                    0 & |\text{{input}}_i| == 0 \\
+                    \frac{{\text{{input}}_i}}{|{\text{{input}}_i}|} & \text{otherwise}
+                    \end{cases}
+
+.. note::
```

Comment:
I think this shouldn't be a note since the function has been added specially for complex numbers. So, I'd suggest to undo these doc changes.

Reply:
What about incorporating the note into a new opening paragraph?

"An extension of torch.sign() to complex tensors. This function is equivalent to torch.sign() for non-complex tensors, and for complex tensors it computes a new tensor whose elements have the same angles as the corresponding elements of :attr:\`input\` and absolute values (i.e. magnitudes) of one."

Then the math expression can follow this paragraph.

Reply:
yeah that will work.